spec/classes/foreman_proxy__spec.rb

```diff
@@ -462,7 +466,8 @@
               '---',
               ':enabled: https',
               ':tftproot: /tftproot',
-              ':tftp_servername: 127.0.1.1'
+              ':tftp_servername: 127.0.1.1',
+              ':system_image_root: /var/lib/foreman-proxy/tftp/system_images}'
```

Comment:
This line is incorrect.
```suggestion
              ':system_image_root: /var/lib/foreman-proxy/tftp/system_images'
```